### PR TITLE
Fix/3489 query config refactor

### PR DESCRIPTION
**Pull request by CPAlejandro**
Hi guys,
In this PR refactor query Config from Angular to React

To prove it,
We enter the new Office 365 section, click on the Panel tab and click on the i at the top left, which will display the panel. It should not give us any error in console.

Close: #3489

### CODE REVIEW

File: public/components/overview/office-panel/office-panel.tsx [hunk 1]
```diff
@@ -16,7 +16,8 @@ import { MainPanel } from '../../../components/common/modules/panel';
 import { withErrorBoundary } from '../../common/hocs';
 import { ModuleStats } from './module-stats';
 import { WzRequest } from '../../../react-services/wz-request';
-import { queryConfig } from '../../../services/query-config';
+//import { queryConfig } from '../../../services/query-config';
```
Comment (eze9252) [inline public/components/overview/office-panel/office-panel.tsx:19]: remove commented import

File: public/components/overview/office-panel/office-panel.tsx [hunk 2]
```diff
@@ -16,7 +16,8 @@ import { MainPanel } from '../../../components/common/modules/panel';
...
+import { queryConfig } from '../../../react-services/query-config';
 import { getCurrentConfig } from '../../../controllers/management/components/management/configuration/utils/wz-fetch';
 // import { getCurrentConfig } from '../../../react-services';
```
Comment (eze9252) [inline public/components/overview/office-panel/office-panel.tsx:22]: remove commented import

**Review by eze9252 — CHANGES_REQUESTED** [review]

File: public/components/overview/office-panel/office-panel.tsx (hunk 1 above)
Comment (asteriscos) [inline public/components/overview/office-panel/office-panel.tsx:19]: Could you please remove the comments to improve code readability?

**Review by asteriscos — APPROVED** [review]
CR: Besides cleaning some comments, looks good to me :heavy_check_mark:

**Pull request merged**